txstratum/protocol.py

```diff
@@ -169,11 +185,10 @@ def get_next_message_id(self) -> int:
     def send_and_track_request(self, method: str, params: Any) -> None:
         """Send a request to the client and track it."""
         msgid = self.get_next_message_id()
+        now = txstratum.time.time()
 
         self.messages_in_transit[msgid] = MessageInTransit(
-            id=msgid,
-            method=method,
-            timeout=txstratum.time.time() + self.MESSAGE_TIMEOUT,
+            id=msgid, method=method, timeout=now + self.MESSAGE_TIMEOUT, created_at=now
```

Comment:
Why not one parameter per line?

Reply:
The linter did it but I can change again

Reply:
I changed it back but the linter complained again